### DISCUSSION TITLE
Require all Scarab Lord Prospects to do the War Effort

### DISCUSSION THREAD

**Discussion by Alaeria-Val** [chromiecraft/chromiecraft#4106] [Suggestions]
As it stands, the war effort is being carried by a handful of people. There are too many people going for scarab lord and not enough people doing the war effort. You guys(Chromiecraft direction team) went against the blizzlike progression by having Scarab Lord quests being able to be accessed prior to war efforts completion. To rectify this, it would be in the servers interest to require ALL Scarab Lord Prospects(those trying to do scarab lord) to have war effort contributions, roughly 70-100 stacks minimum. Preferably 150-200. There is only 5 people afaik actively mass turning in materials, one of them being Resh who has done it on multiple characters from friendly to exalted, who has ultimately given up due to the lack of other people contributing. 

In short: Make leeches who want scarab lord do the war effort by checking for quest completion numbers across all war effort quests, and if below the threshold, they cant ring the gong/get the title.

**Comment by Protandina** [comment]
I have to strongly agree.
By current projections we will get end of war effort within a month.
Some people are concerned because too many scarab lord. 
That is not my concern. 
Any serious player/guild should have them by now.

But people are feed up with current raids and i can see online taking a toll and another month of this is not good.

Even if not to make it a requirement, at least make war effort more worthwhile by increasing the rep gain upon turning in badges.
I did my share of war effort, by wasting all stockpiled items, but if you do not want to spend gold, farming most items is mundane and time consuming task. I wanted rep but i went to do quest as i found that faster and easier.
Or make it so people can get some reasonable money out of it. Even 50 silver from bag would be enough.

Currently there is no incentive and very little gain to get from war effort.
It will not move any faster especially since all major guilds are raid logging.

**Comment by Neversong97** [comment]
Not my problem, where are the people that DIDNT DO Scarab Lord???? THEY DIDNT DO ANYTHING and they want us to open gates for them. Not our problem

**Reply by rheshs** [reply]
well then enjoy waiting 3-4 more months for aq40 and maybe a other 3-4 for tbc this is the only ways to speed up the war effort

**Reply by Alaeria-Val** [reply]
As Protadina said, people who aren't doing Scarab lord have no incentive to do it. Either increase rewards or make scarab lords do the war effort, otherwise no AQ40 for 2 months.

**Reply by Neversong97** [reply]
SO ur telling me that EVEN that Scarab Lords didnt sleep for like 3 days grinding while u guys DIDNT DO ANYTHING, you want us even if we grinded for this shit to open???????????????????????? first of all WHAT THE FUCK, thats same as saying "we gypsies want free money from people who work because its not fair". Second of all "enjoy waiting 3-4" DUDE go OPEN IT GO str8 up if u want it go open it. WE DO NOT CARE.

**Reply by Alaeria-Val** [reply]
The issue is that all the big guilds are delaying this just so they can have their 20-30 scarab lords, rather than it being a race like its supposed to. I genuinely couldnt care less about how long it takes, I will just play other games until its open. Its just overall not healthy for the server to have every guild refuse to do it because "muh scarab lord." We raced the shit out of bug farms to have 4 people ready for scarab lord before reset, but then we find out that its no longer a race because we have to do the war effort.

Edit: Most people outside of the big guilds wont do it either because 1) no actual rewards
2) no one else does it, just a waste of gold turning in.
This shit wont be done in 2 months unless GMs intervene or add in restrictions to the titles, and until then, everyone will continue to raid log as usual.

**Reply by Neversong97** [reply]
where do u think you are playing on? its a SLOW progressive server if its LOW POP and small gold has NO VALIUE. No competition what did u expect gonna happen? everybody here is casual they have family they care less about the game. The best solution for this is to idk this week or next that every turn in counts as x2 and next week counts as 3x etc...

**Reply by Neversong97** [reply]
GMs should care about this more, BUT DO NOT force players who grinded and put effort to get Scarab Lord into new work and custom stuff again. Again DO NOT make custom grinds again its annoying and dumb.

**Reply by Protandina** [reply]
I think many people including Neversong got it wrong.
Going or not going for scarab lords has nothing with war effort. 
Saying non scarab lords do not care about war effort, but scarab lords do is such a silly argument.
Those that are finished with scarab lord, do not care about war effort, why would they ?
Those still farming for it do not want war effort.

Tbh i partially agree with you Neversong. 

1 suggestion was also to give turn ins some xp gains, that way low lvls would actually have incentive to farm war effort.

**Reply by balleny** [reply]
Doing the scarab lord chain was a decision everyone did themselves. No one was forced to grind the bugs.
But, the server got forced to farm for War Effort (and this unfortunately includes everyone, even the PvP players) if they want to progress further (towards TBC, e.g. to play Arena and less boring content). This was the mistake taken here IMHO.

Now, we reached a state
- in which the server will end up with 40+ Scarab Lords (no issue here)
- people are raid logging even more because there is no progression / new content (and yes, if you account for each bracket being roughly 3 months, we have had BWL for 6 already, have AQ20 already for 6 weeks, which equates 3 months if you consider the 3 day reset schedule
- another fast forward expectation of 4+ weeks for War Effort
- followed by likely at least another 6 more weeks of AQ40 (which will, same as BWL, still be run in the lvl 61-64 bracket by many players)

I am more or less wondering, how far we moved past the "Chromiecraft is here to fix AzerothCore" and if we went past some tipping points that cause the population and possible testers to decline heavily.


I would wish strongly for some more transparent communication from the team what the sketched timeline / plan looks like with respect to the next 2 or so brackets. But, I did so repetitively in the past to become aware were to focus for (e.g. will we have a "pre-patch" like phase as on retail which takes away tuning / custom modifications from classic before we advance, or will this e.g. happen in the 69 bracket to reduce impact; will there be an auto-progression through war effort; removal of attunements; flying in tbc, removal of onyxia60, etc.)

**Comment by balleny** [comment]
although i get that chromie wants to progress through the content to get it all fixed etc., I just hope we take the blizzlike route and after 3 months just progress to the next bracket - no matter the state of war effort or not.
maybe even, open AQ40, but let people do, or not do, the war effort farming. If the scarab lord people want it so badly, they will go for it. (and yes, this would be a slap in their face because of the reputation you could earn in AQ40)
